downloadCurrentVersion.js

```diff
@@ -13,7 +15,7 @@ function getVersionList (cb) {
   console.log('Retrieving available version list...');
 
   var mem = new MemoryStream(null, { readable: false });
-  https.get('https://ethereum.github.io/solc-bin/bin/list.json', function (response) {
+  https.get(SOLC_REPO + 'list.json', function (response) {
```

Comment:
```suggestion
  https.get(SOLC_REPO + '/list.json', function (response) {
```
Just in case people forget a trailing slash in the env variable?